helix-rest/src/test/java/org/apache/helix/rest/server/AbstractTestClass.java

```diff
@@ -711,6 +713,62 @@ private void preSetupForCrosszoneParallelInstancesStoppableTest(String clusterNa
     _clusters.add(clusterName);
     _workflowMap.put(clusterName, createWorkflows(clusterName, 3));
   }
+
+  private void preSetupForNonTopoAwareInstancesStoppableTest(String clusterName,
+      List<String> instances) throws Exception {
+    _gSetupTool.addCluster(clusterName, true);
+    ClusterConfig clusterConfig = _configAccessor.getClusterConfig(clusterName);
+    clusterConfig.setFaultZoneType("helixZoneId");
```

Comment:
Based on my reading of the ClusterTopology class, I don't think that the changes in this PR will work in the case that clusterConfig does not have fault zone type set. We need to make sure that stoppable check works for clusters that DON'T have TOPOLOGY, TOPOLOGY_AWARE_ENABLED, and FAULT_ZONE_TYPE configs set.

Please add tests for this case and make the necessary changes to support this.